app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -122,7 +122,18 @@ abstract class RoutePresenter(
         val layoutInflater = LayoutInflater.from(activity)
 
         toast.view = layoutInflater.inflate(R.layout.toast_view, container, false)
-        toast.setGravity(Gravity.FILL_HORIZONTAL or Gravity.BOTTOM, 0, 216)
+
+        val bottomMargin = if (action is ToastNotificationAction.ShowDeleteToast) {
```

Comment:
> Easy: choose the bottom margin based on whether or not the current destination is the ItemList.

```kotlin
val bottomMargin = if (navController.currentDestination?.id == R.id.fragment_item_list) {
```

Reply:
This way we can make all toasts that are displayed on the Item List avoid the `+` button, not just the delete one.

Reply:
Oh that makes perfect sense. Done.